otherlibs/runtime_events/runtime_events.mli

```diff
@@ -617,6 +617,10 @@ val start : unit -> unit
   a set of callbacks to be called for each type of event.
 *)
 
+val path : unit -> string
+(** [path ()] returns the path to the runtime events file, if runtime
+  events are being collected.  *)
```

Comment:
And otherwise, what? (Answer: it fails with a `Failure _` exception, which could be documented.) I would prefer to have the function return a `string option`.

Reply:
I considered that, but thought this was "good enough" for an interface in which other exceptional cases are not documented (e.g. `Runtime_events.create_cursor`) and other functions (e.g. `pause`) silently do nothing if circumstances aren't right (e.g. events aren't enabled). The whole API is perhaps a bit shonky. But that's no excuse to add more shonkiness; I'll tidy this up.

Reply:
I see; with github reviews we don't really see the diff in context, so I tend to be maximally nitpicky by default, apologies.